### DISCUSSION TITLE
What is the reason behind the jQuery only functioning properly on Internet Explorer 8 on this particular page and no other browsers?

### DISCUSSION THREAD

**Discussion by textheader** [jquery/jquery#5550] [Q&A] [answered]
I recently created a webpage utilizing jQuery:

The functionality on the page should change music images to represent different key signatures when switching from 'Higher Key' to 'Lower Key' in the combo box. While this works perfectly in IE8, it's not functioning properly in Safari, Firefox, or Chrome.

What could be causing this issue?

From https://hypertexter.space/questions/what-is-the-reason-behind-the-jquery-only-functioning-properly-on-internet-explorer-8-on-this-particular-page-and-no-other-browsers

Below is the snippet of jQuery code I've implemented:

```
$(document).ready(function () {
    $("#musicKey").change(function (event) {
        if ($("#musicKey").val() * 1) {
            $("img[src*='Low'").each(function (index) {
                $(this).attr("src", $(this).attr("src").replace("Low", "High"));
            });
        }
        else {
            $("img[src*='High'").each(function (index) {
                $(this).attr("src", $(this).attr("src").replace("High", "Low"));
            });
        }
    });
});

```

**Comment by mgol** [comment, marked as answer]
This fits a discussion more so I moved it.

If the site works in IE 8 then you must be using a very old jQuery version as jQuery 2+ versions don't support IE older than 9. Such old jQuery versions are not supported.
